content/10.diagnostics.md

```diff
@@ -518,8 +518,14 @@ During the surges caused by the different SARS-CoV-2 variants between 2020 and 2
-These ongoing changes in testing policy are likely to have a direct effect on how the pandemic is managed moving forward, because SARS-CoV-2 diagnostic tests can be used effectively to slow the spread of the disease only when 1) they are used to share testing results in a timely manner so that they can reasonably be used to approximate the number of infections in the population, and 2) those tests are easily accessible by the general public. 
-In this regard, it is important to mention that the multiple SARS-CoV-2 diagnostic tools we have presented in this paper lack any real benefit if they are difficult to obtain, or if their limited use results in biased data that would lead to ill-informed public health strategies. 
+These ongoing changes in testing policy are likely to have a direct effect on how the pandemic is managed moving forward, because SARS-CoV-2 diagnostic tests can be used effectively to slow the spread of the disease only when 1) they are used to share testing results in a timely manner so that they can reasonably be used to approximate the number of infections in the population, and 2) those tests are easily accessible by the general public.
+
+Children are one segment of the population where the importance of the two aforementioned conditions can be exemplified. 
+This group is particularly vulnerable as there are ongoing challenges with testing in schools, increased COVID-19 mortality rates, and COVID-19-associated orphanhood. 
+In this regard, although there is evidence of the efficacy of routine diagnostic testing to reduce the probability of having infectious students[@doi:10.3389/fpubh.2021.751451;@doi:10.2807/1560-7917.ES.2021.26.34.2100184], as of March of 2022 there is an increasingly number of schools that have stopped or plan to stop contact tracing efforts [@url:https://www.opb.org/article/2022/03/02/oregon-schools-guidance-mask-mandate-end/;@url:https://www.wptv.com/news/education/palm-beach-county-public-schools-to-stop-covid-19-contact-tracing], in line with an announcement made by the CDC where it no longer recommended contact tracing as a strategy to contain the virus [@url:https://www.nytimes.com/live/2022/03/02/world/covid-19-tests-cases-vaccine].
+Considering that an estimated 197 children have died in the US from COVID-19 during the first three months of 2022 [@url:https://www.aap.org/en/pages/2019-novel-coronavirus-covid-19-infections/children-and-covid-19-state-level-data-report/] (compared to 735 deaths in the preceding 20 months of the pandemic [@url: https://www.theguardian.com/world/2022/mar/11/us-child-covid-deaths-omicron-surge]), and that millions of children have been orphaned as a consequence of parent or caregiver death due to COVID-19 [@doi: 10.1016/S2352-4642(22)00005-0], it is likely that these negative outcomes are exacerbated in the upcoming months as a direct result of eliminating contact tracing in schools.
```

Comment:
```suggestion
An estimated 197 children have died in the US from COVID-19 during the first three months of 2022 [@url:https://www.aap.org/en/pages/2019-novel-coronavirus-covid-19-infections/children-and-covid-19-state-level-data-report/], compared to 735 deaths in the preceding 20 months of the pandemic [@url: https://www.theguardian.com/world/2022/mar/11/us-child-covid-deaths-omicron-surge], and millions of children have been orphaned as a consequence of parent or caregiver death due to COVID-19 [@{doi:10.1016/S2352-4642(22)00005-0}].
It is likely that these negative outcomes will be exacerbated in the future as a direct result of eliminating contact tracing in schools.
```
Breaking up the long sentence and fixing the DOI

Reply:
Maybe rephrase the last sentence as follows to highlight testing and to generalize the time frame

"It is likely that these negative outcomes will be exacerbated for the remainder of 2022 as a direct result of eliminating or limited testing capacity in schools."

Reply:
Re-wrote the last sentence

Reply:
OK, I think I've taken care of all the comments, @RLordan or @agitter, let me know what you think and if you believe these changes are good to close the PR.

Reply:
> OK, I think I've taken care of all the comments, @RLordan or @agitter, let me know what you think and if you believe these changes are good to close the PR.

All good with me @aimundo, thanks very much. Reads very well.